internal/kafka/command_topic_produce.go

```diff
@@ -632,3 +645,21 @@ func setSchemaPathRef(schemaString srsdk.SchemaString, dir, subject string, sche
 	}
 	return tempStorePath, referencePathMap, nil
 }
+
+func parseHeaders(headers []string, delimiter string) ([]ckafka.Header, error) {
+	var kafkaHeaders []ckafka.Header
+
+	for _, header := range headers {
+		parts := strings.SplitN(header, delimiter, 2)
+
+		key := strings.TrimSpace(parts[0])
+		if key == "" {
+			return nil, fmt.Errorf(invalidHeadersErrorMsg, delimiter)
+		}
+
+		if len(parts) == 2 {
+			kafkaHeaders = append(kafkaHeaders, ckafka.Header{Key: key, Value: []byte(strings.TrimSpace(parts[1]))})
```

Comment:
Let's try to prevent really long lines.
```suggestion
            kafkaHeader := ckafka.Header{
                Key:   key,
                Value: []byte(strings.TrimSpace(parts[1])),
            }
			kafkaHeaders = append(kafkaHeaders, kafkaHeader)
```

Reply:
✅